tests/callbacks/test_finetuning_callback.py

```diff
@@ -357,24 +358,30 @@ def test_callbacks_restore(tmpdir):
     assert len(callback._internal_optimizer_metadata[0]) == 2
 
     # original parameters
-    assert callback._internal_optimizer_metadata[0][0] == {
+    expected_ = {
```

Comment:
WHy the suffix underscore?

```suggestion
    expected = {
```